gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/idl/gobblin.service.flowconfigs.restspec.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name" : "flowconfigs",
+  "namespace" : "gobblin.service",
+  "path" : "/flowconfigs",
+  "schema" : "gobblin.service.FlowConfig",
+  "doc" : "generated from: gobblin.service.FlowConfigsResource",
```

Comment:
Please add a javadoc to the resource. That should automatically create a doc here.